ceph/nvmeof/nvmeof_gwcli.py

```diff
@@ -70,34 +70,52 @@ def get_subsystems(self):
         """Get all subsystems."""
         return self.run_control_cli("get_subsystems")
 
-    def create_block_device(self, name, image, pool, block_size=None):
+    def get_spdk_nvmf_log_flags_and_level(self):
+        """Get nvmeoF log flags and level."""
+        return self.run_control_cli("get_spdk_nvmf_log_flags_and_level")
+
+    def disable_spdk_nvmf_logs(self):
+        """Disable SPDK NVMeoF logs."""
+        return self.run_control_cli("disable_spdk_nvmf_logs")
+
+    def set_spdk_nvmf_logs(self, **kwargs):
+        """Set SPDK NVMeoF logs."""
+        args = {"flags": True}
+        args = {**args, **kwargs}
+        return self.run_control_cli("set_spdk_nvmf_logs", **args)
+
+    def create_block_device(self, name, image, pool, **kwargs):
         """Create block device using rbd image."""
         args = {"image": image, "pool": pool, "bdev": name}
-        if block_size:
-            args["block-size"] = block_size
+        args = {**args, **kwargs}
+        return self.run_control_cli("create_bdev", **args)
+
+    def resize_block_device(self, bdev, size):
+        """Create block device using rbd image."""
+        args = {"bdev": bdev, "size": size}
         return self.run_control_cli("create_bdev", **args)
```

Comment:
the action parameter is wrong here, it should be `resize_bdev` instead of `create_bdev`